pkg/asset/manifests/infrastructure.go

```diff
@@ -36,6 +37,17 @@ import (
 var (
 	infraCfgFilename           = filepath.Join(manifestDir, "cluster-infrastructure-02-config.yml")
 	cloudControllerUIDFilename = filepath.Join(manifestDir, "cloud-controller-uid-config.yml")
+
+	// convert the installer formatted gcp api names to openshift formatted api names.
+	gcpServiceEndpointNameMap = map[string]configv1.GCPServiceEndpointName{
+		gcp.CloudResourceManagerServiceName: configv1.GCPServiceEndpointNameCloudResource,
+		gcp.ComputeServiceName:              configv1.GCPServiceEndpointNameCompute,
+		gcp.DNSServiceName:                  configv1.GCPServiceEndpointNameDNS,
+		gcp.FileServiceName:                 configv1.GCPServiceEndpointNameFile,
+		gcp.IAMServiceName:                  configv1.GCPServiceEndpointNameIAM,
+		gcp.ServiceUsageServiceName:         configv1.GCPServiceEndpointNameServiceUsage,
+		gcp.StorageServiceName:              configv1.GCPServiceEndpointNameStorage,
```

Comment:
We have `configv1` in `pkg/types`, it seems like we could just that everywhere, and not carry our own type. WDYT?